clientlib/src/main/proto/yelp/nrtsearch/search.proto

```diff
@@ -392,7 +392,8 @@ message SearchResponse {
         double newSnapshotSearcherOpenMs = 7;
         double nrtWaitTimeMs = 8;
         map<string, double> facetTimeMs = 9;
-        double rescoreTimeMs = 10;
+        double rescoreTimeMs = 10 [deprecated = true];
```

Comment:
not sure about deprecated = true here. Does this affect anything? We are still returning rescoreTimeMs right?

Reply:
Reverted change. We will support both `rescoreTimeMs` and `rescorersTimeMs` for now.